src/entry-points/content/AllMediaElementsController.ts

```diff
@@ -57,6 +57,7 @@ export type TelemetryMessage =
     elementLikelyCorsRestricted: boolean,
     elementCurrentSrc?: string,
     createMediaElementSourceCalledForElement: boolean,
+    elementRemainingDuration: number,
```

Comment:
```suggestion
    /**
     * Remember that this could be `Infinity` for live streams,
     * and if `.duration` is otherwise unknown.
     */
    elementRemainingIntrinsicDuration: number,
```

Added a comment and made more explicit what kind of duration it is (intrinsic).